src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -7,6 +7,9 @@
     position: fixed;
     right: 0;
     z-index: 99999;
+    -webkit-border-top-left-radius: 4px;
+    -moz-border-radius-topleft: 4px;
```

Comment:
According to [the official Firefox reference](https://developer.mozilla.org/es/docs/Web/CSS/border-top-left-radius), the `-moz-border-radius-topleft` property is not needed since Firefox 12 (latest version is 42).